### ISSUE TITLE
Applications - Oracle Cloud

### ISSUE THREAD

**Issue by stefangrosaru** [goauthentik/authentik#3912]
**Describe your question/**
Is Oracle Cloud still supported? Tried using the provided documentation, but it seems to be outdated and uses oidc. I tried to configure a SAML integration and got as far as being able to log into Oracle Cloud via authentik but I can't seem to manage to get the groups/permissions applied to the logged in user.

**Version and Deployment (please complete the following information):**
 - authentik version: 2022.10.0
 - Deployment: helm

**Comment by BeryJu** [comment]
The oracle docs are using OIDC on purpose and work as intended, not sure what you mean with groups/permissions? I dont use Oracle cloud all that much so I haven't looked into mapping any further attributes than the defaults

**Comment by stefangrosaru** [comment]
As you can see from the screenshot attached, there is no OIDC option only SAML
<img width="1408" alt="image" src="https://user-images.githubusercontent.com/48685107/206520633-c8f0be30-e064-4fcc-a699-c712ca6f5c4b.png">

**Comment by stefangrosaru** [comment]
This is off topic but do you know if this works on a free account?

Can't find the menu in the screenshot, that's why I'm asking.

**Comment by BeryJu** [comment]
Yes, I have all of this setup on a free account too

**Comment by stefangrosaru** [comment]
That's weird I cannot find this menu anywhere.
<img width="1041" alt="image" src="https://user-images.githubusercontent.com/48685107/206796968-296db0cb-dfc1-496e-99c1-7bda66e27adb.png">

**Comment by BeryJu** [comment]
Try accessing it via https://cloud.oracle.com/identity/domains/

**Comment by stefangrosaru** [comment]
Tried that, but no luck
<img width="1041" alt="image" src="https://user-images.githubusercontent.com/48685107/207264356-eaefa75a-01d5-4fa4-90f7-0a18ff6651de.png">

**Comment by browningluke** [comment]
@stefangrosaru I managed to get SSO working with SAML, since the 'Domains' section also disappeared for me.

1. I created a property mapping like so:
<img width="1095" alt="image" src="https://github.com/goauthentik/authentik/assets/24992944/e4351be6-5c13-489c-83c6-4d1286eeb01a">

```
oci-saml-groups (this can be whatever)

https://auth.oraclecloud.com/saml/claims/groupName

GroupName

for group in request.user.ak_groups.all():
    yield group.name
```

2. Assigned it to my OCI SAML provider.

Authentik now correctly passes group information to OCI, and my configured OCI group mappings work.

Hope this helps!
